### ISSUE TITLE
labels and legends for plominofields for wcag compliance.

### ISSUE THREAD

**Issue by djay** [plomino/Plomino#335]
Currently plomino forms fail WCAG 2.0 accessibility. One main reason is lack of labels and legends.

My suggested way to fix this is a new style/class called "Plomino Label".

For example

``` html
<h3>Pools</h3>
<p><span class="plominoFieldClass">display_pools</span></p>
```

could be enhanced with

``` html
<h3>Pools</h3>
<p><span class="plominoFieldClass">display_pools</span></p>
<p><span class="plominoLabelClass">display_pools: This is the Label for pools</span></p>
```

optionally you can skip the label text and it will use the field title. e.g.

``` html
<h3>Pools</h3>
<p><span class="plominoFieldClass">display_pools</span></p>
<p><span class="plominoLabelClass">display_pools</span></p>
```

With regard to checkboxes and radio buttons it's a requirement to have a fieldset and legend. see http://html.cita.illinois.edu/nav/form/checkbox/index.php?example=6

This could be done in the same way. e.g. if display_pools was a checkbox or radio button list then the plominoLabelClass will become a legend and the whole result will be wrapped in a fieldset. If the list is a combobox then it will generate a label as previous examples.

If no plominoLabelClass is used then existing behaviour is the same which makes this change backwards compatible.
